### PR TITLE
Add Dashboard UI

### PR DESCRIPTION
**Pull request by Taaseen-Ali**
This PR:
- adds the metrics panel to dashboard
- connects dashboard to pull data from image proxy
- adds a route in the image proxy to fetch all moderation requests. 
- adds the ability for the image proxy to serve the dashboard ui (when running the dashboard can be accessed via `localhost:3000/admin`)
- adds the dashboard to docker build process

Screenshots of the updated dashboard screens:
image
image
image

### CODE REVIEW

File: dashboard-ui/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# cra-template-tailwindcss-typescript
```
Comment (bhaskarkishore) [inline dashboard-ui/README.md:1]: Let's remove this generic page

File: dashboard-ui/src/config.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "proxyURL": "http://localhost:3000",
+  "apikey": "e67b309d4587e57e93070ba49e5d33aaa09a9e6da4af9654f0708df6c738aabd"
```
Comment (bhaskarkishore) [inline dashboard-ui/src/config.json:3]: Are these intended to be development defaults?

File: proxy.conf [hunk 3]
```diff
@@ -57,7 +57,7 @@
     # Cache configuration
     "cache_config": {
         # Options are MemoryBoundedLruCache, DiskCache, None
-        "cache_type": "None",
+        "cache_type": "MemoryBoundedLruCache",
```
Comment (bhaskarkishore) [inline proxy.conf:60]: Let's revert this change.

File: dashboard-ui/src/config.json (hunk 2 above)
Comment (Taaseen-Ali) [reply dashboard-ui/src/config.json:3]: they'll also need to be used in production.
